arm-software/linux/docs/GettingStarted.md

```diff
@@ -0,0 +1,322 @@
+## Getting Started
+
+This document describes how to get started with the already installed Arm
+Toolchain for Linux. Here you will find out how to use it to compile a source
+code into an executable binary.
+
+### Looking at the toolchain
+
+The default installation site of Arm Toolchain For Linux is the
+`/opt/arm/arm-toolchain-for-linux` directory. It contains a complete set of LLVM
```

Comment:
Isn't this going to have a version number? My ACfL is installed to `/opt/arm/arm-linux-compiler-24.04_Ubuntu-22.04`

Reply:
As I witnessed it with our current rpms, there is no version number, it's up to the package manager to keep track on it. This can cause concerns when it comes to installing multiple versions on one machine, but from what I remember, the admins of such machines should do manual installation that would prevent a conflict on installation target directories. How it is going to be, I guess we still need to work out.

We could ask @garyjcarroll for confirmation.